cluster/charts/rook-ceph/templates/deployment.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
     operator: rook
     storage-backend: ceph
     chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    rook-version: {{ .Values.image.tag }}
```

Comment:
The `rook-version` label should come from `github.com/rook/rook/pkg/version.Version` applied by the Rook operator and not from the name of the image as shown here. Remove this line.